### ISSUE TITLE
Color Palette

### ISSUE THREAD

**Issue by Finn-Wyll** [COS301-SE-2026/AutoFlow-Trade-Simulator#15]
A defined set of primary, secondary, and accent colours, including their
HEX or RGB values. These colours should be selected with accessibility in mind, ensuring
sufficient contrast for readability.

**Comment by u24641342** [comment]
Adding this
